PMMain/Midas/MidasGUI/MainGUI.cpp

```diff
@@ -34,18 +38,23 @@ MainGUI::MainGUI(MidasThread *mainThread, int deadZoneRad)
 
     layout->setAlignment(infoIndicator, Qt::AlignRight);
     layout->setStretchFactor(infoIndicator, 0);
-    
+    layout->setAlignment(distanceDisplayer, Qt::AlignRight);
+    layout->setStretchFactor(distanceDisplayer, 0);
```

Comment:
nice job getting this in here. Likely will cause merge conflict - but will be an easy fix